cmd/api-errors.go

```diff
@@ -574,6 +575,12 @@ var errorCodeResponse = map[APIErrorCode]APIError{
 		Description:    "Server not initialized, please try again.",
 		HTTPStatusCode: http.StatusServiceUnavailable,
 	},
+	ErrOpNotAcceptable: {
+		Code:           "XMinioOpNotAcceptable",
+		Description:    "Requested operation is not accepted due to unsatisfied conditions.",
```

Comment:
```
MethodNotAllowed	The specified method is not allowed against this resource.	405 Method Not Allowed	Client
``` 

http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html  - is perhaps better here.